plugin.yml

```diff
@@ -1,9 +1,14 @@
 name: SimpleCape
-api: [3.0.0]
 main: padreon\SimpleCape\Main
+website: https://github.com/padreon/SimpleCape
+description: This is a SimpleCape plugin for Servers
+api: 3.0.0
```

Comment:
Might want to keep that in `[]` to allow more version to be added. Also, version strings should be enclosed in quotes

Reply:
Okay